### PR TITLE
Add a script to run the license script and update the golden file in one step.

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

**Comment by Hixie** [comment]
As far as I can tell, this doesn't actually generate the only file that matters, the sky_engine/LICENSE file.

File: tools/licenses/README [hunk 1]
```diff
@@ -1,8 +1,5 @@
-To update the sky_engine LICENSE file, run the following in this directory:
+To update the sky_engine LICENSE file, run the following from the build root:
 
 ```
-pub get
-gclient sync
-dart lib/main.dart ../../.. > ../../sky/packages/sky_engine/LICENSE
-dart --checked lib/main.dart ../../.. > ../../travis/licenses.golden
+./flutter/tools/licenses/update_licenses.sh
```
Comment (goderbauer) [inline tools/licenses/README]: There is one more step: If the diff actually indicated that an entire license was added or removed, you need to run the license script again in release mode (it has a flag) and pipe the output into `sky/packages/sky_engine/LICENSE`

Comment (chinmaygarde) [inline tools/licenses/README]: How does the diff indicate that? I just pasted the script we used to update the files for my last PR. Let me try adding another file and see what happens.

Comment (Hixie) [inline tools/licenses/README]: I mean, you have to read the diff and see...

**Pull request closed without merging**